feg/cloud/go/services/feg_relay/feg_relay/main.go

```diff
@@ -42,11 +42,16 @@ func main() {
 		glog.Fatalf("Failed to create FegToGwRelayServer: %v", err)
 		return
 	}
+
+	// Register responders FEG -> AGW
+	lteprotos.RegisterSessionProxyResponderServer(srv.GrpcServer, servicer)
+	lteprotos.RegisterAbortSessionResponderServer(srv.GrpcServer, servicer)
+	protos.RegisterS8ProxyResponderServer(srv.GrpcServer, servicer)
```

Comment:
thius `RegisterS8ProxyResponderServer` was missing

The rest has been repositioned